BitwardenShared/UI/Platform/Application/AppProcessor.swift

```diff
@@ -323,3 +324,88 @@ extension AppProcessor: SyncServiceDelegate {
         }
     }
 }
+
+// MARK: - Fido2 credentials
+
+public extension AppProcessor {
+    /// Provides a Fido2 credential for a passkey request.
+    /// - Parameter passkeyRequest: Request to get the credential.
+    @available(iOSApplicationExtension 17.0, *)
+    func provideFido2Credential( // swiftlint:disable:this function_body_length
+        for passkeyRequest: ASPasskeyCredentialRequest
+    ) async throws -> ASPasskeyAssertionCredential {
```

Comment:
🎨 What do you think about moving most of this body out of the `AppProcessor`? I put the `provideCredential` in `DefaultAutofillCredentialService` and I thought it was nice to keep a bulk of the logic out of the `AppProcessor`.

Reply:
I can but if it's ok I'd prefer to do that in #749 where more logic is there and so there is less merging problems afterwards.

Reply:
Yep, no problem!